go.mod

```
@@ -206,7 +206,7 @@ replace github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
 replace github.com/rjeczalik/notify => github.com/rjeczalik/notify v0.9.3
 
 // use cometbft
-replace github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.27
+replace github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.29
```

```suggestion
replace github.com/tendermint/tendermint => github.com/cometbft/cometbft v0.34.30
```
we can bump to the latest patch version
https://github.com/cometbft/cometbft/blob/v0.34.30/CHANGELOG.md#v03430